packages/1password/data_stream/audit_events/_dev/test/system/test-default-config.yml

```
@@ -0,0 +1,8 @@
+input: httpjson
+service: 1password_eventsapi_mock
+vars:
+  url: http://{{Hostname}}:{{Port}}
+  token: --token--
+  preserve_original_event: true
+data_stream:
+  vars: ~
```

There is a new capability in `System Testing` for `httpjson` type input to assert on the number of events ingested in whole after pagination and/or cursors. May be it is worth using it

This helps to catch errors in future if some events get lost due to some changes in the pipeline etc.,

Reference [here](https://github.com/elastic/integrations/blob/main/packages/google_workspace/data_stream/admin/_dev/test/system/test-default-config.yml#L23-L24)

Reference PR - #5326 

```suggestion
  vars: ~
 assert:
   hit_count : xxx
```